fibers/web/server.scm

```diff
@@ -176,6 +173,35 @@ on the procedure being called at any particular time."
               ((0) (memq 'keep-alive (response-connection response)))))
            (else #f)))))
 
+(define* (make-chunked-output-port* port #:key (keep-alive? #f)
+                                    (buffering 1200))
+  "This is a variant of make-chunked-output-port from (web http) which
+importantly sends a \r\n right at the end of the output.  Without
+this, some programs will not finish reading from the server."
+
+  (define (%put-string s)
+    (unless (string-null? s)
+      (let* ((bv (string->bytevector s "ISO-8859-1"))
```

Comment:
Could use a comment on why ISO-8859-1 (probably just a reference to relevant RFC). Do you mean ISO-8859-1, or just ASCII?

Reply:
I've removed this procedure now.